src/lints/static_unsafe_added.ron

```diff
@@ -0,0 +1,51 @@
+SemverQuery(
+    id: "static_unsafe_added",
```

Comment:
I appreciate you naming this while keeping in mind consistency with existing lints. Our lint naming isn't particularly consistent in general, and in this case I think we may be better off breaking consistency to avoid confusion. To trigger this lint, it's not the case that someone added `unsafe` but that they replaced `safe` with `unsafe`.

So perhaps a name like `static_became_unsafe` would be best? If you agree, make sure to rename both the `id` and the file name, as well as the test crate, snapshot file, and `query.rs` entry.

In the future, I imagine we might rename other lints to be consistent with this one, instead of vice versa.